app/views/admin/shared/_adminnavbar.html.erb

```diff
@@ -1,6 +1,6 @@
 <nav class="navbar fixed-top navbar-expand-lg navbar-dark bg-dark">
   <div class="container-fluid">
-    <a class="navbar-brand" href="/">RERA Admin Dashboard</a>
+    <a class="navbar-brand" href="/admin">RERA Admin Dashboard</a>
```

Comment:
I would rather use `link_to` helper offered by Rails :)

Reply:
Updated! Thanks for hint ;)